### ISSUE TITLE
Display real line number in pages

### ISSUE THREAD

**Issue by haiqi96** [y-scope/yscope-log-viewer#7]
### Request

The current design of log viewer display the first line number as 1 on all pages. This is confusing to me at the beginning, because logs in different pages look very similar and I was referring to the line number to see if I have moved to another page. 

### Possible implementation

Can we let the log viewer display the actual line number in the original file?

**Comment by vishalpalaniappan** [comment]
If we display the real line number, then the space taken up for the line number column will start to increase. In many cases, there are millions of log events, in my opinion this would be worse. Right now, we display the log event that is selected in the status bar, that indicates our progression through the log and line number indicates where we are in the current page.

I might be wrong, I'm going to do some tests to see how it would look if we kept the original line number across pages. I plan to have infinite scrolling eventually, need to think about how line number would work in this situation.

**Comment by Henry8192** [comment]
I agree that this looks confusing at the beginning. Considering the number of log events could be large, displaying real log event number on the side is not a good idea. The real log event number is currently displayed in the url parameter `logEventNum` as well as in the status bar. By changing that `logEventNum` in the url, the log viewer is able to jump to the designated log event. Considering the pros and cons, it's better to stay in this way.
